app/controllers/api/v1/restaurants_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V1::RestaurantsController < ApplicationController
+
+  def index
+    render json: { restaurant: Restaurant.all.sample }
+  end
+
+  def create
+    body = JSON.parse(request.body.read)
+    restaurant = Restaurant.new(body)
+    if restaurant.save
+      render json: { restaurant: restaurant }
+    end
+  end
+
+  private
+
+  def restaurant_params(body)
+    body.require(:restaurant).permit(:name, :address, :city, :state, :zip, :dollar_value,
+      :description, :picture, :phone_number, :website_url, :user)
+  end
```

Comment:
I love seeing `strong_params`, but I'm not sure you're ever actually utilizing them in this controller -- you appear to just be passing the entire unfiltered hash assigned to `body` (on line 8) to the `.new()` method. 😞

Reply:
Hi Eben, we worked with Brianna for a few to try and pass the request body into the strong params but were unsuccessful. I'm not sure we had the correct syntax. How can we pass the request body into the strong params?

Reply:
You would probably have to transform the top-level key from `body` to `restaurant`. Which you could do inside the controller, given your current implementation.